### PR TITLE
Add logic

### PR DESCRIPTION
**Pull request by Boku52500**
- Installed bigjs with npm i big-js.
- Added logic folder
- Inserted calculate.js and operate.js files in logic folder.
- Added logic in calculator.jsx to comply with calculate.js and operate.js according to requirements.
- Made calculator functional.
- Added some styling.
- Fixed linters.
- Updated README file.

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Boku52500, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that has not utilized the new [readme template](https://github.com/microverseinc/readme-template) by Microverse for documentation. Kindly note that henceforth, it is required to utilize for your readme in order to receive a review. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            <Button handleClick={this.addToInput}>.</Button>
+            <Button handleClick={() => this.handleEqual()}>=</Button>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+export default Calculator;
```
Comment (MaganaAsiati) [inline src/components/Calculator.jsx]: You did a great job by developing you're on code to make the calculator work but I  advised you to use the code already-developed, math logic needed to make work the Calculator component. as it was required in the project requirement 

- the project must have an operate.js file copied from [this one](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js). :+1: 
-  The project must have a calculate.js file copied from [this one](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/calculate.js). and at the end Copy the files calculate.js and operate.js into a logic/ directory in your project. :+1: 
Analyze the files calculate.js and operate.js, be sure you understand what they do (note: you need to add [big.js](https://www.npmjs.com/package/big-js) as a dependency).
- Import the files in your Calculator component. :+1: 
- Implement the event handlers you need to use the math logic from calculate.js and operate.js in your React component. :+1: 
- Practice what you have learned until now: use state and props, pass props to child components, use lifecycle methods, and lift the state up. :+1: 
- Test your app in the browser. It should be able to perform math operations. :+1:

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @Boku52500 , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

- Kindly make sure when opening your app in the browser there is non linter error as it shows in the picture below please fix that it will make your app look clean and professional :+1: 
 
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [x] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [x] [OPTIONAL] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Calculator.jsx (hunk 1 above)
Comment (Boku52500) [reply src/components/Calculator.jsx]: @MaganaAsiati Thank you for reviewing my code. I am now going to go through the components I have improved after your suggestions.

- Added operate.js file :heavy_check_mark:
- Added calculate.js file :heavy_check_mark:
- Imported the files in Calculator component. :heavy_check_mark:
- Implemented the event handlers in React component and used the math logic from calculate.js and operate.js :heavy_check_mark:
- Used state, proprs, passed props to child components, used lifecycle methods and lifted the state up. :heavy_check_mark:
- Tested my app in the browser. Everything is working in order. :heavy_check_mark:

**Review by Gopxfs — APPROVED** [review]
# 🟢 STATUS: APPROVED 🟢

## Hello @Boku52500 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Linters are passing successfully.
✔️ Your PR is looking very professional 😉.
✔️ Imported the logic files to create your calculator. Well done!
✔️ Perfect usage of React components, props & events.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**